skills/lark-base/references/workflow-steps/common-types-and-refs.md

```
@@ -0,0 +1,401 @@
+# Workflow common types and refs
+
+### ValueInfo
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)